pkg/api/types.go

```diff
@@ -305,13 +305,19 @@ const (
 	ReleaseChannelStable    ReleaseChannel = "stable"
 	ReleaseChannelFast      ReleaseChannel = "fast"
 	ReleaseChannelCandidate ReleaseChannel = "candidate"
+
+	// BuildRootImageFileName is the name of the file that contains the build root images
+	// pullspec.
+	BuildRootImageFileName = ".build_root_image"
```

Comment:
Could we go with a `.ci-operator.yaml` or something to future-proof this?

Reply:
Sure, so make it a struct (which for now will only have this one option)?

Reply:
done